sympy/printing/tests/test_repr.py

```diff
@@ -335,3 +335,7 @@ def test_set():
     assert srepr(s) == "set()"
     s = {x, y}
     assert srepr(s) in ("{Symbol('x'), Symbol('y')}", "{Symbol('y'), Symbol('x')}")
+
+
+def test_AppliedPredicate():
+    sT(Q.even(Symbol('z')), "AppliedPredicate(Q.even, Symbol('z'))")
```

Comment:
Thanks for this test - can you also add one to `test_str.py`, something like `assert sstr(Q.even(Symbol('z'))) == "..."`?